Options.py

```diff
@@ -1115,6 +1125,17 @@ def verify(self, world: typing.Type[World], player_name: str, plando_options: "P
             link.setdefault("link_replacement", None)
 
 
+class Removed(FreeText):
+    """This Option has been Removed."""
+    default = ""
+    visibility = Visibility.none
+
+    def __init__(self, value: str):
+        if value:
+            raise Exception("Option removed, please update your options file.")
```

Comment:
Could this error message be a property on the option? I could see maintainers using this wanting to give an indication of recourse, like "use this other option / functionality instead"

Reply:
yes, you override `__init__`